discovery/discovery_test.go

```diff
@@ -1193,18 +1195,80 @@ func TestOrchestratorPool_GetOrchestrators(t *testing.T) {
 
 }
 
+func TestOrchestratorPool_GetOrchestrators_SuspendedOrchs(t *testing.T) {
+	assert := assert.New(t)
+	require := require.New(t)
+	addresses := stringsToURIs([]string{"https://127.0.0.1:8936", "https://127.0.0.1:8937", "https://127.0.0.1:8938"})
+
+	orchCb := func() error { return nil }
+	oldOrchInfo := serverGetOrchInfo
+	defer func() { serverGetOrchInfo = oldOrchInfo }()
```

Comment:
Looks like CI is complaining about a data race here:

```
==================
WARNING: DATA RACE
Write at 0x000002cf3c28 by goroutine 191:
  github.com/livepeer/go-livepeer/discovery.TestOrchestratorPool_GetOrchestrators_SuspendedOrchs.func2()
      /build/discovery/discovery_test.go:1205 +0x3a
  github.com/livepeer/go-livepeer/discovery.TestOrchestratorPool_GetOrchestrators_SuspendedOrchs()
      /build/discovery/discovery_test.go:1258 +0xf06
  testing.tRunner()
      /usr/lib/go-1.13/src/testing/testing.go:909 +0x199

Previous read at 0x000002cf3c28 by goroutine 182:
  github.com/livepeer/go-livepeer/discovery.(*orchestratorPool).GetOrchestrators.func1()
      /build/discovery/discovery.go:56 +0xa0

Goroutine 191 (running) created at:
  testing.(*T).Run()
      /usr/lib/go-1.13/src/testing/testing.go:960 +0x651
  testing.runTests.func1()
      /usr/lib/go-1.13/src/testing/testing.go:1202 +0xa6
  testing.tRunner()
      /usr/lib/go-1.13/src/testing/testing.go:909 +0x199
  testing.runTests()
      /usr/lib/go-1.13/src/testing/testing.go:1200 +0x521
  testing.(*M).Run()
      /usr/lib/go-1.13/src/testing/testing.go:1117 +0x2ff
  main.main()
      _testmain.go:90 +0x223

Goroutine 182 (finished) created at:
  github.com/livepeer/go-livepeer/discovery.(*orchestratorPool).GetOrchestrators()
      /build/discovery/discovery.go:74 +0x470
  github.com/livepeer/go-livepeer/discovery.TestOrchestratorPool_GetOrchestrators_SuspendedOrchs()
      /build/discovery/discovery_test.go:1221 +0x3cc
  testing.tRunner()
      /usr/lib/go-1.13/src/testing/testing.go:909 +0x199
==================
--- FAIL: TestOrchestratorPool_GetOrchestrators_SuspendedOrchs (0.00s)
    testing.go:853: race detected during execution of test
```

Reply:
fixed in 6d3937b89aac4c754ba97b6ffd844d0035ee8f38